README.md

```diff
@@ -23,7 +23,44 @@ and the [Checker Framework Language Server](https://github.com/eisopux/checker-f
 
 A JDK is required, i.e. `JAVA_HOME` needs to be properly set. JDK versions 8, 9, and 11 are supported. Node.js and npm are also required.
 
-### Configuration
+## Build from source
```

Comment:
For a normal user, the "Configuration" section is more important than building from source and the developer notes.
So I think we should move this new material back down in the page to where the developer section is. I would put "Building from source" as a subsection in the developer notes.
Moving the changes back down will also make it easier to see what the differences are. We can move the section around afterwards, if you think.

Reply:
Got it, I added a hyperlink to build from source section, I guess that would be helpful.